centreon/www/include/configuration/configObject/contactgroup/DB-Func.php

```
@@ -94,59 +110,64 @@ function multipleContactGroupInDB($contactGroups = [], $nbrDup = [])
+    );
+    $contactSelectStmt = $pearDB->prepare(
+        'SELECT DISTINCT `cgcr`.`contact_contact_id` FROM `contactgroup_contact_relation` `cgcr`'
+        . ' WHERE `cgcr`.`contactgroup_cg_id` = :cgId'
+    );
+    $contactInsertStmt = $pearDB->prepare(
+        'INSERT INTO `contactgroup_contact_relation` VALUES (:cct, :maxId)'
+    );
 
-        $row = $dbResult->fetch();
-        $row['cg_id'] = null;
+    foreach ($contactGroups as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)